src/style.css

```
@@ -0,0 +1,119 @@
+@font-face {
+  font-family: "Avenir-Book", sans-serif;
+  font-weight: 300;
+  font-style: normal;
+  src: url("./fonts/Avenir-Book.ttf");
+}
+
+@font-face {
+  font-family: "Avenir-Heavy", sans-serif;
```

```suggestion
@font-face {
  font-family: "Avenir";
  font-weight: 300;
  font-style: normal;
  src: url("./fonts/Avenir-Book.ttf");
}

@font-face {
  font-family: "Avenir";
```
It's one font-family just  with different font-weights